bundle/Form/Type/SearchTagsType.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Netgen\TagsBundle\Form\Type;
+
+use Symfony\Component\Form\Extension\Core\Type\SearchType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\Length;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+class SearchTagsType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        parent::configureOptions($resolver);
+
+        $resolver->setDefaults([
+            'method' => 'GET',
+            'csrf_protection' => false,
+        ]);
+    }
+
+    public function getBlockPrefix(): string
+    {
+        return '';
```

Comment:
No need for implementing this method.

Reply:
If I leave the generated block prefix, the request path looks like this `/ngadminui/tags/admin/search?search_tags%5BsearchTerm%5D=test` and the HTML search form in the top menu (https://github.com/netgen/NetgenAdminUIBundle/pull/38) doesn't work.

Reply:
I'm not really sure what one has to do with the other, but this is definitelly not the solution to the problem :)

TagsBundle is non-legacy and should not cater to legacy specific issue in legacy admin.

Reply:
This is a GET form, we don't really need nor have any use for the block prefix here. And it prevents the other form from working for no reason, even if it's legacy admin... What do you suggest instead?

Reply:
I'm still not sure how resetting the block prefix helps here and what is the connection between the block prefix and this being a `GET` form? Can't you just replicate the markup generated here in legacy admin, whatever that markup may be?

The block prefix is used to facilitate easier overriding of form styling and should definitelly not be set to an empty value, but at least be left at a default value.

Reply:
What I meant by this being a get form is it only serves to build the query string, really, and should be as plain and simple as possible. But ok, what you said makes sense. I'll update the other form then.

Reply:
If you wanted this to be as simple as possible, there was no need for Symfony Form in the first place :D

Look at how search form/controller was implemented in `netgen/media-site`. Similar thing could've been replicated here.

Reply:
Removed the form class completely, added a HTML form. Done in https://github.com/netgen/TagsBundle/commit/f4a8ea03a3e319d60e3f799889a2fd0b12598249